### ISSUE TITLE
Add Benchmarking

### ISSUE THREAD

**Issue by TheAngryByrd** [TheAngryByrd/MiniScaffold#256]
**Is your feature request related to a problem? Please describe.**

Add [benchmarksDotNet](https://github.com/dotnet/BenchmarkDotNet) and use [github-action-benchmark](https://github.com/benchmark-action/github-action-benchmark) to display them

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by artem-burashnikov** [comment]
Hello. Could you please elaborate on what is expected to be added to templates?

**Comment by TheAngryByrd** [comment]
What specifically are you looking for in elaboration? Issue points to using the `benchmarkDotnet` template for benchmarks and then using `github-action-benchmark` to display them.

**Comment by artem-burashnikov** [comment]
As I am still very inexperienced, I would like to clarify: you mean that benchmark library to be added to generated templates and a workflow job that catches up any benchmarks that may by defined by user later on?

**Comment by TheAngryByrd** [comment]
> benchmark library to be added to generated templates

Yep! Probably under a `benchmarks/` folder similar to how there's `src/` or `tests/`.

> a workflow job that catches up any benchmarks that may by defined by user later on

Yep! The workflow should be preconfigured to pick up the templated benchmark.  Should also be easy to amend when someone creates new benchmarks.
